lib/tasks/docker.rb

```diff
@@ -1,28 +0,0 @@
-DOCKER_NAME = ENV["DOCKER_NAME"] || "ondemand-dev"
-
-namespace :docker do
-    desc "Build Docker container"
-    task :build => ["package:test_container"]
```

Comment:
I didn't want to remove this initially, but `package:test_container` doesn't exist anymore.